libraries/app/include/graphene/app/application.hpp

```diff
@@ -38,7 +38,7 @@ namespace graphene { namespace app {
    class application_options
    {
       public:
-         bool enable_subscribe_to_all = false;
+         bool enable_subscribe_to_all = true;
```

Comment:
Don't change this one to `true`. 

Sorry, my above comment was not clear enough. Expected behavior:
* this option is disabled by default (if not specified in config.ini nor startup parameters);
* if a user specified it in parameters or config.ini, if without `=X` (only `--enable-subscribe-to-all` but not `--enable-subscribe-to-all=true`), consider the value is `true`.